client/src/pages/commonPage/footer/index.tsx

```diff
@@ -1,16 +1,56 @@
-import { Box, Container } from '@mui/material';
+import { Box, Container, Grid, Typography, } from '@mui/material';
+import logo2 from 'assets/images/logo2.png';
 import styles from './styles';
 import React from 'react';
 
 function Footer(props: any) {
   const { container } = styles;
+  const { link } = styles;
+  const { title } = styles;
   return (
-    <Box sx={{ background: 'black' }}>
+    <Box sx={{ background: '#212121' }}>
       <Container maxWidth='xl' sx={container}>
         <Box component='p'>
-            Hello
+            <img style={{ width: 164, height: 46, }} src={logo2} alt="logo" />
```

Comment:
1. <Box component='img'> tag should be used, avoid to use 'styles'
    -Fixed: <Box component='img' width={164} height={46} src={logo2}>
2. image source should be named explicitly, example: 'shopLogo'